common/web/tsconfig.kmw-main-base.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../web/tsconfig.base.json"
+}
```

Comment:
Given this file is just an include of another file, is there really any reason to keep it? (Don't say 'for the future' please!)

Reply:
The main reason I had it in there was in case there's a need to relocate where we keep the "true" include.  If so, I can just swap the definitions and not have to update every tsconfig that depends on the two.

If the locations I've selected look to be fine moving forward, I don't mind nixing this one.  It just provides a single point-of-update should the relative locations change.  I'd never gotten fully comfortable with the locations that were selected, partly because it feels awkward (to me) for `/common/web` to dip into _anything_ from the `/web` section.